src/analysis/monotone-analyzer-impl.h

```diff
@@ -96,12 +68,21 @@ inline void MonotoneCFGAnalyzer<L, TxFn>::collectResults() {
 template<Lattice L, TransferFunction TxFn>
 inline void MonotoneCFGAnalyzer<L, TxFn>::print(std::ostream& os) {
```

Comment:
Can you explain why this print function instead of a << overload? Does it have to do with Concepts?

Reply:
No particular reason. The original author probably thought `print` was a nicer interface than `<<`, and that seems like a reasonable opinion to me. A `<<` overload would have worked just as well. Print methods generally have a slight advantage that they are easier to call from a debugger, although I'm not sure whether it's easy to get the ostream to pass into this one from the debugger. `toString` methods are generally the easiest for debugging.